<launch>
  <node name="image_process" pkg="TOMO_final" type="Transform_coordinate.py" output="screen"/>
  <node name="move_group_python_interface_tutorial" pkg="TOMO_final" type="pick_up_blisters.py" output="screen"/>

</launch>
